std/stdio.d

```diff
@@ -2226,6 +2298,42 @@ $(XREF file,readText)
         assert(blc.front is blc.front);
     }
 
+    /**
+    Parse from the file handle one line at a time into a record tuple.
+
+    Range primitives may throw $(D StdioException) on I/O error.
+
+    Params:
+        file = file handle to parse from
+        format = tuple record $(XREF_PACK std/format/formattedRead, format)
+
+    Returns:
+        Returns an input range set up to parse from the file handle one line
+        at a time into a record tuple.
```

Comment:
With the above text for the description, this becomes just "The input range". (The word Returns is already in the section title).